cmd/subnetcmd/deploy.go

```diff
@@ -678,3 +692,45 @@ func PrintDeployResults(chain string, subnetID ids.ID, blockchainID ids.ID, isFu
 	table.Render()
 	return nil
 }
+
+// Has side effects and will set the version of avalanchego in the package-level avagoVersion variable.
+func checkForInvalidDeployAndSetAvagoVersion(network nw.StatusChecker, configuredRPCVersion int) error {
+	// get current network
+	networkRunning := true
+	runningAvagoVersion, runningRPCVersion, err := network.GetCurrentNetworkVersion()
+	if err != nil {
+		// not actually an error, network just not running
+		app.Log.Debug(err.Error())
+		networkRunning = false
+	}
+
+	// RPC Version was made available in the info API in avalanchego version v1.9.2. For prior versions,
+	// we will need to skip this check.
+	skipRPCCheck := false
+	if semver.Compare(runningAvagoVersion, "v1.9.2") == -1 {
+		skipRPCCheck = true
+	}
+
+	if networkRunning {
+		if avagoVersion == "latest" {
```

Comment:
probably will me more clear by using a more descriptive name, not just avagoVersion

Reply:
renamed